lib/repository/repository.rb

```diff
@@ -29,6 +40,10 @@ def following_table
   end
 
   def time_before(number_days)
-    Time.now.to_i - 86400 * number_days
+    Time.now - number_days * DAY + last_execution_duration
+  end
+
+  def last_execution_duration
+    5 * MINUTE  # approx fixed
```

Comment:
Can you remove the comment, please? I think it's clear enough with the method name.